pkg/cmd/controller/pipelinerunner.go

```diff
@@ -144,6 +145,51 @@ func (o *PipelineRunnerOptions) Run() error {
 	return nil
 }
 
+func (o *PipelineRunnerOptions) configure() error {
+	err := o.setLogLevel()
+	if err != nil {
+		return errors.Wrap(err, "unable to set log level")
+	}
+
+	logger.Infof("meta pipeline enabled: %t", o.UseMetaPipeline)
+	if o.UseMetaPipeline {
+		logger.Info("creating meta pipeline client")
+		client, err := metapipeline.NewMetaPipelineClient()
+		if err != nil {
+			return errors.Wrap(err, "unable to create meta pipeline client")
+		}
+
+		client.SetServiceAccount(o.ServiceAccount)
+		client.SetDefaultImage(o.MetaPipelineImage)
+
+		o.metaPipelineClient = client
+	} else {
+		if !o.NoGitCredentialsInit {
+			err := o.InitGitConfigAndUser()
+			if err != nil {
+				return err
+			}
+
+			err = o.stepGitCredentials()
+			if err != nil {
+				logger.Warn(err.Error())
+			}
+		}
+	}
+	return nil
+}
+
+func (o *PipelineRunnerOptions) setLogLevel() error {
+	level := os.Getenv("JX_LOG_LEVEL")
```

Comment:
Shouldn't this take into account also some CLI log level options? At least we should document this env variable.

Reply:
Thing is with the CLI options I cannot fully control the log level anymore. This is captured in issue #5080 and #4581. 

So yes, I kind of behind the back introduce `JX_LOG_LEVEL`. Preferably this should be done as part of #5080 and be handled in the log package. 

If we agree that it makes sense to have this env variable, I can remove this from this PR and create a new for for #5080. WDYT?

Reply:
sgm!